_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,294 @@
+---
+name: Iframes with identical accessible names serve equivalent purpose
+
+description: |
+ This rule checks that iframes with identical accessible names serve an equivalent purpose
```

Comment:
If you do some of the changes suggested below, changing the description might also make sense, e.g. to: 

> This rule checks that iframes with identical accessible names embed the same resource or equivalent resources.